server/lib/safe-json.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+// via https://gitlab.com/gitterHQ/webapp/-/blob/615c78d0b0a314c2c9e4098b8d2ba0471d16961b/modules/templates/lib/safe-json.js
+function safeJson(string) {
+  if (!string) return string;
+  // From http://benalpert.com/2012/08/03/preventing-xss-json.html
+  return string.replace(/<\//g, '<\\/');
+}
```

Comment:
This is from Gitter.

You can see how we use it in a similar way to inline a JSON payload into the HTML page to generate `window.troupeContext`: https://gitlab.com/gitterHQ/webapp/-/blob/615c78d0b0a314c2c9e4098b8d2ba0471d16961b/modules/templates/lib/hbs-helpers.js#L47-56